projects/eval_scannet/scannet/extract_posed_images.py

```diff
@@ -0,0 +1,217 @@
+# Modified from
```

Comment:
license header at top of file
+ we should move to src/home_robot/home_robot/datasets/scannet.py i think